curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e8fe3a6f022a05a04675.md

```diff
@@ -238,7 +335,7 @@ select {
 ```js
 const calorieCounter = document.getElementById('calorie-counter');
 const budgetNumberInput = document.getElementById('budget');
-const entryDropdown = document.getElementById('entry-dropdown');
+const entryDropdownElement = document.getElementById('entry-dropdown');
```

Comment:
why these changes to the seed?
are they reflected in previous/next steps?

Reply:
I didn't realize I changed the seed. That was a complete accident. Too many copy and replaces.